test/Microsoft.ML.Tests/TextLoaderTests.cs

```diff
@@ -803,6 +803,41 @@ public void TestTextLoaderKeyTypeBackCompat()
             }
         }
 
+        [Fact]
+        public void TestCommaAsDecimalMarker()
+        {
+            string dataPath = GetDataPath("iris_decimal_marker_as_comma.txt");
+
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext(seed: 1);
+            var reader = new TextLoader(mlContext, new TextLoader.Options()
+            {
+                Columns = new[]
+                        {
+                            new TextLoader.Column("Label", DataKind.Single, 0),
+                            new TextLoader.Column("Features", DataKind.Single, new [] { new TextLoader.Range(1, 4) }),
```

Comment:
Since you're already testing here both the cases of loading a single column, and multiple columns as a feature vector, I think this is currently correctly testing both the ProcessOne and ProcessVec path on the TextLoader.Parser.

If anything, I'd also add a test to load DataKind.Double, just to make sure that your changes also work from TX to R8. (this can be done by modifying this test with an InlineData attribute with a "useDouble" bool parameter that switches between reading this as Single or as Double).